src/user/screens/profile.screen.js

```diff
@@ -57,12 +57,16 @@ const mapDispatchToProps = dispatch =>
     dispatch
   );
 
-const styles = StyleSheet.create({
-  listTitle: {
+const BioListItem = styled(ListItem).attrs({
```

Comment:
This is the same style as in the auth profile screen.
Another +1 to separate styles from components ;)